Update enablemfa.md (#3205)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/mfa/main/ga/enablemfa.md

> 1. From the Admin Console, select **Security** from the main menu and then **Authenticators**.

Reviewer: ```suggestion
1. In the Admin Console, go to **Security** > **Authenticators**.
```

> 2. Click the **Add Authenticator** button, select **Google Authenticator** (OTP).

Reviewer: ```suggestion
2. Click **Add Authenticator**, and then click **Add** on the Google Authenticator tile.
```

> 3. Click the Add button associated with the service.

Reviewer: ```suggestion
3. Click **Add** to add the service. 
```
